File: Day 35-36/randomNumbers.c

#include <stdio.h>
#include <math.h>

int throwDice(int faces);

int main()
{

    printf("How many times do you want to throw the dice? \n");
    int n;
    scanf("%d", &n);

    int i;
    for( i=0; i<n; i++ ){

        throwDice(6);

    }

    throwDice(10);
    throwDice(12);
    throwDice(4);

    return 0;
}

int throwDice(int faces){

    int number = rand()%faces + 1;

    printf("%d \n", number);

    return number;

}

File: Day 35-36/randomNumbers.js
function random(min, max){

    var res;
    res = Math.floor(Math.random() * (max - min +1)) + min;
    return res;

}

function throwDices(faces){

    let number = random(1, faces)

    console.log( number )
    
}

throwDices(6)
throwDices(10)
throwDices(12)